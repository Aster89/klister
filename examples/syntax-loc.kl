#lang kernel

(define here 'here)

(define there 'there)

(example here)

(example (replace-loc there here))

