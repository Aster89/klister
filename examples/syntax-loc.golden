#[syntax-loc.kl:3.15-3.19]<here> : Syntax
#[syntax-loc.kl:5.16-5.21]<here> : Syntax
